feathr_project/feathr/client.py

```diff
@@ -569,6 +571,22 @@ def _get_sql_config_str(self):
             """.format(JDBC_TABLE=table, JDBC_USER=user, JDBC_PASSWORD=password, JDBC_DRIVER = driver, JDBC_AUTH_FLAG = auth_flag, JDBC_TOKEN = token)
         return config_str
 
+    def _get_snowflake_config_str(self):
+        """Construct the Snowflake config string for jdbc. The dbtable (query), user, password and other parameters can be set via
+        environment variables."""
```

Comment:
update the doc. Seems only url, user and role are set via yaml. Password is set through env.